TrackletAlgorithm/Constants.h

```diff
@@ -45,6 +45,7 @@ constexpr unsigned kMaxProcOffset(const module::type m) {
 constexpr int kNBits_MemAddr = 7;
 constexpr int kMemDepth = (1<<kNBits_MemAddr);
 constexpr int kNBits_BX = 3;
+constexpr unsigned int kNMatchEngines=4;
```

Comment:
I'm not very familiar with the match processor but why is kNMatchEngines set to 4? I see that you use kNMatchEngines when calling the VMStubMECM memories as the NCOPY parameter, but there's only one copy of each VMStubMECM in the testvectors?

Reply:
@aryd responded further down, but I'll summarize it here, too. We use 4 MEUs to process the Match Engine steps in parallel. These use 4 identical copies of the same memory. This wastes some memory, but prevents pipelining issues when two modules are trying to read from the same memory at the same time.